packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -94,13 +93,50 @@ class ShorebirdEnv {
   }
 
   /// The `shorebird.yaml` file for this project.
-  File getShorebirdYamlFile() {
-    return File(p.join(Directory.current.path, 'shorebird.yaml'));
+  File getShorebirdYamlFile({Directory? cwd}) {
```

Comment:
I suspect having it optional might end up being a footgun. 🤷

Reply:
I refactored to make it required